DataManagementSystem/Client/DataManager.py

```diff
@@ -616,7 +616,7 @@ def __isSEInList( self, seName, seList ):
         seSet.add( res['Value'] )
     return self.__getSERealName( seName ).get( 'Value' ) in seSet
 
-  def __replicate( self, lfn, destSE, sourceSE = '', destPath = '', localCache = '' ):
+  def __replicate( self, lfn, destSEName, sourceSEName = '', destPath = '', localCache = '' ):
```

Comment:
Since it is a named argument, are you sure that it is not called with it ? It should be at least...
Anyway, why would you change that ?